app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class UsersController < ApplicationController
   def index
-    @users = User.order(:id).page(params[:page])
+    @users = User.order(:id).page(params[:page]).with_attached_avatar
```

Comment:
特に決まりはないですが、orderやpageは最後の方に書くことが多い気がします。

```suggestion
    @users = User.with_attached_avatar.order(:id).page(params[:page])
```

Reply:
確かに感覚的にもorderなどは後ろに書く方がしっくりきます、ありがとうございます！